### ISSUE TITLE
Unable to reproduce results

### ISSUE THREAD

**Issue by balabala-h** [youqingxiaozhua/APViT#11]
Thank you for your open source contribution！

I took out the model architecture and trained it using my own training code. And, I followed the optimization function, optimizer, and optimization strategy in your code, but I did not get enough good results, with the highest accuracy of only 86%.
[train epoch 98] loss: 0.057, acc: 0.982: 100%|█████████████████████| 95/95 [01:12<00:00,  1.31it/s]
[valid epoch 98] loss: 0.564, acc: 0.863: 100%|█████████████████████| 23/23 [00:18<00:00,  1.24it/s]
[train epoch 99] loss: 0.056, acc: 0.981: 100%|█████████████████████| 95/95 [01:12<00:00,  1.32it/s]
[valid epoch 99] loss: 0.565, acc: 0.863: 100%|█████████████████████| 23/23 [00:18<00:00,  1.25it/s]

I don't know where the problem lies. And I dont understa what's that mean
![image](https://user-images.githubusercontent.com/73676575/236249995-2b0863da-a6a6-4e7f-a0ad-add23528ebfb.png)

**Comment by youqingxiaozhua** [comment]
86% is too low for such a deep model. There must be something misaligned.

The `paramwise_cfg` you mentioned is used to set the weight decay of position embeddings and cls_token of ViT to zero, same as the original ViT model:

https://github.com/huggingface/pytorch-image-models/blob/3d05c0e86f2f4c57bf495468aa3f8a7d3487c986/timm/models/vision_transformer.py#LL523C31-L523C31

Hope this is helpful for you.

**Comment by balabala-h** [comment]
thank you for your reply!
 I am now at 90.5% accuracy by fixing some wrong operations.

**Comment by youqingxiaozhua** [comment]
Glad to hear that. Before submitting the codes, I run the released code and get an accuracy of about 91.8%, so there might be still a 1% gap.

**Comment by balabala-h** [comment]
Thank  your , I will try to reproduce this result.
